### ISSUE TITLE
How could I get Toolbar text from Keyboard

### ISSUE THREAD

**Issue by MKelman** [hackiftekhar/IQKeyboardManager#385]
The IQKeyboardManager has been working great except for now when I need to use it with a 3rd party picker. I am also using custom table view cells and it seems to interfere with what I am trying to work with. How could I get that text from the keyboard in another class. 

I tried something like : let toolbar:IQToolbar = IQToolbar() and then calling toolbar.title! but that doesn't seem to work well since title is set to nil

tldr; Looking at the screenshot, how could I receive the word "Password" in another class? 

![screen shot 2015-12-17 at 11 37 15 pm 1](https://cloud.githubusercontent.com/assets/10063610/11889409/06a7fa72-a517-11e5-8355-24c18104d596.png)

**Comment by MKelman** [comment]
I reread the documentation and found that textField.isAskingCanBecomeFirstResponder == false is my answer. Thank you again for this!

**Comment by hackiftekhar** [comment]
The text which is showing there is extracted using textfield.placeholder. So you can try to get the textfield placeholder text, which will give you the text.
If you are trying to build a custom toolbar like this, then please take a look at IQUIView+IQKeyboardToolbar.h, you'll fine some methods, and you can call those methods to any textfield in order to add your custom toolbar.
